### DISCUSSION TITLE
USPS Postal Money Order payment method

### DISCUSSION THREAD

**Discussion by cparke2** [bisq-network/bisq#7244] [General]
The Bisq wiki contains a page regarding how to prepare and send a USPS Money order for fiat payment.
[https://bisq.wiki/US_Postal_Money_Order](https://bisq.wiki/US_Postal_Money_Order)

I've made two recent trades with USPS Money Order, once as a BTC Buyer and once as a BTC Seller, and I must profess, there are some issues with the things mentioned on the wiki page vs. what others are doing vs. reality.  I think the guidance there should be revised.

**Money Order Purchaser's Name/Address:** Bisq wiki page is inconsistent on its face on this subject.  First, the wiki page authorizes the use of a fake name for the BTC Buyer and discusses two options for a fake address. Then, it instructs in a highighted box to use the BTC Seller's name/address as the Payer and Payee on the USPS Money Order ("anonymous option", see more about that below), as if that is the only acceptable way to do it! The discussion does not provide any guidance, when using fake name/address option on the USPMO, if that **_same fake name/address_** should also be entered into the Bisq account information for the BTC buyer in the Bisq application? Furthermore, nowhere does it discuss that using the actual BTC buyer's name and address is also acceptable, which is really the right way to do it.

**Customer Receipt:** The USPS Money Order comes in two parts; a check-like section and a tear-off "Customer Receipt" section. Post offices also issue an ordinary white receipt slip which includes the USPMO serial number on it and how it was paid for. The money order receipt is typically used to show proof of value in case the money order is claimed by the buyer to be lost, stolen, or damaged. Ordinarily, the money order purchaser (BTC Buyer) fully completes the money order, including the customer receipt portion, and retains it. However, if using a fake name/address, or the BTC seller's name as the money order purchaser, this receipt is probably not going to be of any use to them, so maybe they should include it in the package with the USPMO sent to the BTC Seller? Some BTC Sellers also do not immediately cash or deposit USPMO's the receive, since USPMO's technically never expire, but a USPMO that remains long-term outstanding could be fraudulently claimed as lost/stolen by the BTC Buyer/USPMO Purchaser and stopped and replaced, so it would be preferable for BTC seller to also obtain all receipts associated with the USPMO purchase.

**Anonymous Buyer option:** When I went to USPS to cash a USPS Money Order where Payer and Payee are the same (both me), the office also wanted to see the "Customer Receipt" section of the USPMO. Local postmaster told me that is the official policy for cashing when a Money Order has been written out this way, and if the USPMO was actually bought by someone else, need to send it back to them to be replaced. The way the postal service see it, the person cashing out the money order is also the money order buyer, so the **"Customer Receipt"** may also serve as a form of identification that you are the buyer. If a USPMO purchaser is following the rules, the only legitimate reason to fill out a USPMO in this way is because they bought the money order by mistake or changed their mind and now are cashing it to get their money back. In my case, the local postmaster took two forms of ID when I told them that someone else had sent this money order to me and I was not the purchaser, and they cashed it anyway but mentioned that they weren't supposed to. Frankly, the Bisq policy allowing money orders to be sent anonymously liked this puts the BTC seller in an awkward position; I imagine such transactions could be flagged as suspicious for money laundering, etc. and launch a confidential investigation, in which case I might be obligated to disclose to the authorities at a later time all information that I have available about who sent me the money order and the full details of the Bisq transaction stored on my peer. So there really needs to be some revision to this policy, probably making this payment variation available only at the **_BTC Seller's option_**, and a discussion added to the wiki page about sending the "Customer Receipt" section with the money order in such cases. However, I really think this _**rare exception**_ in the Bisq trade policy just should be totally rescinded and disallowed; Buyer needs to provide some name/address different than the seller on USPMO, and that information must match what is entered on the Bisq account information in order to be credited towards the trade in the event of a dispute. Problem solved! If someone really wants anonymity, a better way would be to just send the entire **_blank money order_** (including the Customer Receipt portion) to the BTC seller and otherwise follow the "Cash By Mail" trade protocol. 

**Check Image option:** For another USPS Money Order transaction as a BTC buyer, the seller allowed me to just send a check image rather than actual original money order, to save time and postage. Thanks to Check 21 Act, that should work, as he would just scan the money order image with a check deposit app, and the banking system will attempt to collect it. However, the method of sending the check image needs to be properly sanctioned, as anyone intercepting the email with that USPMO check image attachment could attempt to fraudulently deposit it. For that reason, since Bisq and Bisq2 (where this trade actually happened) don't seem to support secure images between traders, I used Protonmail to send the image in an encrypted email message, which supposedly is safe and secure.  Sellers accepting money orders in this way, as well as anyone in general depositing USPS Money Orders into a bank account rather than cashing them at the post office, need to realize that they are waiving several important security features and protections of the USPS Money Order payment method by doing so, as it will be converted into an ordinary check and processed through the Federal Reserve System instead.

**Money Order Verification:** While an authentic USPS Money Order is a generally safe and secure payment method, that does not mean it is a risk-free instrument. Check fraud and counterfeiting is still a real thing, and USPS website has a page discussing all the security features embedded in the USPS Money Order document (see "How to Spot a Fake" in [Sending Money Orders](https://www.usps.com/shop/money-orders.htm)). USPS furthermore has a phone number 1-866-459-7822 and website [Check Money Order Status](https://tools.usps.com/money-orders.htm) to validate the USPS Money Order's authenticity. As a result, Money Orders received should be checked carefully by the Seller for printing discrepancies, verified online that the money order is still valid and has not been already cashed, and preferably also actually cash it out at USPS, all before the clicking "Confirm Payment" within the Bisq application. After all, BTC buyer funds have not actually been received by the seller until they've actually been collected. This is particularly important as we live in an age where "double presentment" of a check is easy to do through personal mobile app deposits, and if that is accomplished by a fraudulent BTC buyer, it can cause the BTC seller to lose money to a scammer (both banks may initially credit the deposits, but ultimately USPS will dispute the multiple payments, and the scammer's bank account could already be closed at that point).

**Trade Period** Currently, Bisq only allows **8 days** for a USPS Money Order trade to complete. That is inadequate. In my case, the trade was opened on Monday, September 2, 2024 (Labor Day, a U.S. holiday), next day BTC buyer bought the USPS Money Order and mailed it by Certified Mail once USPS had opened, but it was not even delivered to my P.O. Box until September 10 (a week later), when the trade period had expired. BTC Buyer immediately requested dispute before I even had a chance to pick up the letter at the post office! The timing could have been even worse; had the trade opened on Saturday, August 31 (beginning of the long weekend), it would have been 10 days into the trade until the money order was even delivered. As mentioned in the previous section, even after receiving the Money Order by mail, I still needed more time after to verify its authenticity and cash it out. Many post offices do not hold enough cash anymore to pay out money orders, so the BTC seller has to either visit several post offices to find one that's has enough money on hand, or, as in my case, visit the post office in the morning and request that they order the cash from the bank to pay out the Money Order later that day or the next day.  As a result, I really feel that a more appropriate normal trade period for the USPS Money Order payment payment method is **14 days**.

In conclusion, while the USPS Money Order is a great payment option for buying BTC, the Bisq trading rules and guidance on the Bisq wiki and in the Bisq application probably needs to be revised to address these important issues and concerns.

**Comment by pazza83** [comment]
Hi @cparke2 thanks for documenting this info. I will take a look at the wiki and your post and make changes / come back to you with more questions. Many thanks
